client/src/pages/Statistics.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import styled from '@emotion/styled';
-import WaveIcon from '../components/assets/wave.js';
+import WaveIcon from '../components/icons/wave.js';
```

Comment:
Is `wave.js` a component? If so, please rename the file to `Wave.js`.
See https://stackoverflow.com/questions/10523849/changing-capitalization-of-filenames-in-git for changing capitalization.